samples/samples-js/AddProductWithMultiplePrimaryColumnsAndIdentity/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = async function (context, request) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const testProduct = {
```

Comment:
You should be parsing the name and cost from the parameters like the C# function